lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -1312,6 +1312,15 @@ def _test_unicode_data(self, text, mock_request):
         self.assertEqual(response_data["discussion_data"][0]["title"], text)
         self.assertEqual(response_data["discussion_data"][0]["body"], text)
 
+    def _test_invalid_course_id(self):
+        """ Asserts that Http404 is raised when the course id is not valid. """
+        request = RequestFactory().get("dummy_url")
```

Comment:
@mushtaqak , I think this is a case of premature optimization. Let's go with this for now, and we can abstract out if we find ourselves continuing to repeat ourselves.